src/applications/bmqstoragetool/m_bmqstoragetool_parameters.cpp

```diff
@@ -413,13 +416,16 @@ Parameters::Parameters(const CommandLineArguments& arguments,
     }
     else if (!arguments.d_seqNumLt.empty() || !arguments.d_seqNumGt.empty()) {
         bmqu::MemOutStream errorDescr(allocator);
+        bool               success = false;
```

Comment:
`success` is not checked after, is it okay?

Reply:
Validation is  done in `CommandLineArguments::validate()`. Added assert.